src/Storage.gs

```diff
@@ -0,0 +1,115 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Contains classes used to persist data and access it.
```

Comment:
Instead of:
`Contains classes used to persist data and access it.`

Say something like:
`Contains utility methods for an in-memory key-value store.`

Storage does persist to a file system, blob files, etc. It's an in-memory KVS.

Reply:
It's actually not all in memory.